src/app/core/services/label.service.ts

```diff
@@ -157,6 +183,28 @@ export class LabelService {
     }
   }
 
+  /**
+   * Returns the definition of the label by searching a list of
+   * all available labels.
+   * @param labelValue: the label's value (e.g Low/ Medium/ High / ...).
+   * @param labelCategory: the label's category (e.g Type/ Severity / ...).
+   */
+  getLabelDefinition(labelValue: string, labelCategory: string): string {
+    if (labelValue === '' || labelCategory === '') {
+      return ''; // not sure of returning empty string as definition here, might have better alternatives
```

Comment:
In my honest opinion, it would be better to return it as `null` here in this case. 

Also, it would be good to add extra guard clauses for `null` values given as arguments in `getLabelDefinition` here. What are you thoughts? @ducbinh2611

Reply:
I agree with Sean, unless there are benefits to using an empty string in this case? @ducbinh2611 Let us know if there are any!😀

Reply:
@seanlowjk @kkangs0226 Sure! I initially put it as an empty string because I just want to be consistent with the other labels which have a definition.

But given the fact that we might put the HTML template of each label definition (if any) on the repo to be fetched during runtime, I think returning it as `null` is perfectly fine too.